src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/HelpWithFeesDetails.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+
+import java.math.BigDecimal;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class HelpWithFeesDetails {
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING)
+    private BigDecimal remissionAmount;
+    private FeeType hwfFeeType;
```

Comment:
Is this required here? this has been moved to caseData

Reply:
for my pr yes, I can merge after changes are done